andela/ah-infinity-stones-frontend
# #161966862 Setup React Boilerplate

#### What does this PR do?
- Setup an empty react project

#### Description of Task to be completed?
- Setup an empty react project
- install and configure eslint

#### How should this be manually tested?
1. git clone -b ch-react-setup-161966862 https://github.com/andela/ah-infinity-stones-frontend.git  and $ cd ah-infinity-stones-frontend
2. Start the app by running npm start
	this will run the app in the development mode. 	Open http://localhost:3000 to view it in the browser.
	The page will reload if you make edits. 	You will also see any lint errors in the console.


3. Test the app by running npm test
	this will Launch the test runner in the interactive watch mode.

#### What are the relevant pivotal tracker stories?
[#161966862](https://www.pivotaltracker.com/n/projects/2225605/stories/161966862)

#### Screenshots
![image](https://user-images.githubusercontent.com/33648833/50780723-bd3fd080-12b4-11e9-8e7c-8a17e25509fb.png)
![image](https://user-images.githubusercontent.com/33648833/50780779-e19bad00-12b4-11e9-8e32-91c0871fd394.png)
